README.md

```diff
@@ -136,17 +136,17 @@ Ember-3d-nav uses Sass for styles. The default styles for the nav are stored in
 
 ```scss
 $ember-3d-nav-defaults: (
-  'body-bg': #485274,
+  'nav-items': 5,
   'nav-container-bg': #000,
   'nav-bg': darken(#485274, 10%),
-  'text-color': #fff,
   'is-fixed-nav-bg': darken(#485274, 5%),
-  'nav-items': 5,
+  'nav-text-color': #fff,
   'nav-item-hover': darken(#485274, 15%),
   'nav-item-selected': darken(#485274, 20%),
   'nav-height-small': 80px,
   'nav-height-medium': 170px,
-  'nav-marker': #e35746,
+  'nav-trigger-color': #fff,
```

Comment:
Super picky, but could we alphabetize these, so it's easier to find the one you are looking for?